.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build the candy-board project
+
+on:
+  push
+  # push:
+  #   branches:
+  #   - main
+  # release:
+  #   types: [created, published]
+
+jobs:
+  build_project:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 14
+    - run: yarn add vite
+    - run: yarn build
+    - uses: actions/upload-artifact@v3
+      with:
+        name: build-artifact
+        path: build/
```

Comment:
I forgot to mention, the result of the build command is folder called `dist` :wink:

Reply:
Done